src/transformers/tape.js

```diff
@@ -78,8 +78,6 @@ const tPropertiesUnsupported = new Set([
     'notDeepLooseEqual',
     'notLooseEqual',
     'notLooseEquals',
-
-    'skip',
```

Comment:
The t property is still unsupported (i.e. `t.skip()`)... So we shouldn't remove this.